### DISCUSSION TITLE
[Fundamentos do C#] [Formatando moedas]

### DISCUSSION THREAD

**Discussion by joaobrambati** [balta-io/forum#535] [Tira Dúvidas] [answered]
Olá, usei a formatação de moedas e funcionou funcionou no real, libra e dólar. Mas no euro, não exibiu, apareceu uma interrogação no lugar do símbolo do euro. 

Cheguei até instalar o idioma espanhol e coloquei "es-ES", mas persistiu o erro. Creio que o problema possa estar ligado ao símbolo.

**Comment by henriqueweege** [comment, marked as answer]
Oi, @joaobrambati ! Tudo bem? 
Teria como postar teu código pra gente dar uma olhada?
Valeeeu!

**Reply by joaobrambati** [reply]
![Captura de tela 2024-04-01 112031](https://github.com/balta-io/forum/assets/102823316/4d01ead7-ebab-4156-8af2-8dd556ad6e41)
Segue imagem Henrique.

Dólar, real e libra aparecem corretamente.
![Captura de tela 2024-04-01 064038](https://github.com/balta-io/forum/assets/102823316/1cd77456-430d-48d9-ad5b-ba8eb078c2e3)
![Captura de tela 2024-04-01 064001](https://github.com/balta-io/forum/assets/102823316/35101c69-43da-48d9-b502-eea144414b49)

**Reply by henriqueweege** [reply]
@joaobrambati,
Se você setar o Console para ler o Output com UTF-8 ele passa a interpretar o caractere da maneira correta. Basta adicionar a linha abaixo.
![image](https://github.com/balta-io/forum/assets/96916801/16c78e13-73a9-47e0-ac0a-5934321c00e3)
Se quiser deixar no mesmo padrão que os outros outputs, a resposta [desse issue do StackOverflow](https://stackoverflow.com/questions/47612295/how-to-display-the-euro-symbol-at-the-front-of-the-number-when-using-cultureinfo) resolve o problema!

Valeeeeu!

**Reply by joaobrambati** [reply]
Cara, muito obrigado, funcionou. Era uma coisa tão simples kkk mas que queria saber o pq não estava funcionando

**Reply by henriqueweege** [reply]
@joaobrambati nada, não!
É só que o encoding padrão da Console que não continha esse caractere, aí não soube o que te mostrar. Quando você diz para a Console usar UTF-8, ela passa a saber interpretar o símbolo  e aí mostra certo ;)
